### DISCUSSION TITLE
archery无法连接开启tls认证的redis实例

### DISCUSSION THREAD

**Discussion by YanXiaoxiao0101** [hhyo/Archery#2061] [General]
### 重现步骤

1.添加开启tls认证的redis实例，超时无法连接；
2.未找到redis配置tls的选项；

### 预期外的结果

测试实例连接时候：
<img width="139" alt="image" src="https://user-images.githubusercontent.com/73461479/220064559-ce468205-a18c-423b-bb5c-c67e746e4340.png">

### 版本

1.9.1

### 部署方式

Docker

### 是否还有其他可以辅助定位问题的信息？比如数据库版本等

redis 版本5.0

**Comment by hhyo** [comment]
暂时不支持
